api/src/domain/user/updateOperatingPhase.test.ts

```diff
@@ -84,20 +118,40 @@ describe("updateOperatingPhase", () => {
           operatingPhase: "NEEDS_TO_FORM",
           legalStructureId: "general-partnership",
         }),
-        taskProgress: { [formationTaskId]: "IN_PROGRESS" },
+        taskProgress: { [formationTaskId]: "IN_PROGRESS", [businessStructureTaskId]: "COMPLETED" },
       });
       expect(updateOperatingPhase(userData).profileData.operatingPhase).toBe("NEEDS_TO_REGISTER_FOR_TAXES");
     });
+
+    it("updates status back to NEEDS_BUSINESS_STRUCTURE if business structure task is not completed", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          businessPersona: "STARTING",
+          operatingPhase: "NEEDS_TO_FORM",
+          legalStructureId: "limited-liability-company",
```

Comment:
should we either have another test for trade name, or do a it.each or a randomizer here?